openc3/lib/openc3/packets/packet.rb

```diff
@@ -556,6 +556,9 @@ def get_item(name)
     #   as Strings. :RAW values will match their data_type. :CONVERTED values
     #   can be any type.
     def read_item(item, value_type = :CONVERTED, buffer = @buffer, given_raw = nil)
+      unless value_type.is_a?(Symbol)
+        raise ArgumentError, "Second argument value_type must be :RAW, :CONVERTED, :FORMATTED, or :WITH_UNITS"
+      end
```

Comment:
I don't get this fix.  Shouldn't the case value_type below have just handled it?

Reply:
Nevermind, I see it was using value_type in the below error.  Let's still move this down into the else case of the when statement to prevent a comparison every single call.